src/Address.h

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2017-2022 Trust Wallet.
```

Comment:
I would keep this AnyAddress class implementation inside TWAnyAddress.cpp.

Reply:
I decided to leave as-is for better readibility.